clang/include/clang/Basic/arm_sme.td

```diff
@@ -740,6 +740,11 @@ let SMETargetGuard = "sme2" in {
   def SVLUTI4_LANE_ZT_X2 : Inst<"svluti4_lane_zt_{d}_x2", "2.di[i", "cUcsUsiUibhf", MergeNone, "aarch64_sme_luti4_lane_zt_x2", [IsStreaming, IsInZT0], [ImmCheck<0, ImmCheck0_0>, ImmCheck<2, ImmCheck0_3>]>;
 }
 
+// FDOT
```

Comment:
There are also variants for `_za32`, ([4-way FDOT](https://developer.arm.com/documentation/ddi0602/2024-09/SME-Instructions/FDOT--4-way--multiple-and-single-vector---Multi-vector-8-bit-floating-point-dot-product-by-vector-to-single-precision-?lang=en)) - would you mind adding them as part of this work?

Reply:
Sure, will do.

Reply:
Now added.